climatetree/frontendfinal
# Story mod delete

### Change type:
- New feature

### Description:
- Admins and moderators can now delete stories
- Delete button will not show up for normal users
- Moved comments to own line in stories
- Add underline to delete button on hover

### Steps to Verify This PR
- You will have to be a moderator or admin to view the delete story button. If you are a regular user, you can verify that the delete button does not show up.

### Note
At the time of submission, the stories service will return 500 error regardless of whether a story was deleted or not.

### Example of story card
@theodoreandrew I can only see the delete comment button here because I'm a moderator user.
![image](https://user-images.githubusercontent.com/29155905/79024473-e0040900-7b37-11ea-9c33-6448c4a41dbe.png)
